Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -233,6 +233,10 @@ class TabTrayController: UIViewController {
     var settingsButton: UIButton!
     var collectionViewTransitionSnapshot: UIView?
 
+    var isPrivateMode: Bool {
+        return privateMode
+    }
```

Comment:
I think you could use `private(set)` on the existing `privateMode` if you wanted.